src/components/UserInputs.js

```diff
@@ -21,9 +21,7 @@ const UserInputs = () => {
           console.log(`hello`);
           navigate('/Dashboard',
           
-     )
-
-     }
+     )}
```

Comment:
While we're fixing the formatting here, why not replace lines 22-26 (now 22-24) with the following?
```
          navigate('/Dashboard');
     }
```